reactor-core/src/main/java/reactor/core/publisher/FluxWindowPredicate.java

```diff
@@ -288,7 +295,7 @@ public Object scanUnsafe(Attr key) {
 
 		@Override
 		public Stream<? extends Scannable> inners() {
-			return window == null ? Stream.empty() : Stream.of(window);
+			return Stream.of(window);
```

Comment:
🐞 the "rebase" commit somehow changed that line. couldn't find a parallel PR where this changed, and I think it is a definite mistake (`window` can definitely be `null`). Change back to `return window == null ? Stream.empty() : Stream.of(window)`